silkworm/db/snapshots/config.cpp

```diff
@@ -60,4 +63,16 @@ BlockNum Config::compute_max_block() {
     return max_block > 0 ? max_block - 1 : 0;
 }
 
+void Config::remove_unsupported_snapshots() {
+    constexpr std::array UNSUPPORTED_SNAPSHOT_NAME_TOKENS = {
```

Comment:
In terms of code style we follow Google C++ Style Guidelines, where the naming convention for constants is [this](https://google.github.io/styleguide/cppguide.html#Constant_Names). Hence, this one should read:
```
kUnsupportedSnapshotNameTokens
```